packages/test-utils/src/testUtils/mockApiRegistry.ts

```diff
@@ -15,12 +15,13 @@
  */
 
 import { ApiTestRegistry } from '@backstage/core-api';
-import { MockErrorApi } from './apis';
+import { MockErrorApi, MockStorageApi } from './apis';
 
 export function createMockApiRegistry(): ApiTestRegistry {
   const registry = new ApiTestRegistry();
 
   registry.register(MockErrorApi.factory);
+  registry.register(MockStorageApi.factory);
```

Comment:
Question unrelated to this PR, what do you think about this as a pattern for all APIs?

Reply:
Sure, at least as a convention, looks good